pages/linux/pdbedit.md

```diff
@@ -0,0 +1,21 @@
+# pdbedit
+
+> Directly edit the Samba user database.
+> Can do more fancy things than `smbpasswd`.
```

Comment:
Can we instead put a See also:
```suggestion
> Similar to `smbpasswd`.
```

Reply:
`pdbedit` provides a superset of features to `smbpasswd`. I'm okay with less casual language but then I'm not sure what's the best way to word it.

Reply:
```
> For simple user add/remove/password, you can also use `smbpasswd`.
```
I suppose this is somewhat more descriptive.